tests/test_function_set.py

```diff
@@ -12,6 +65,9 @@ def setUp(self):
             b = Linear(3, 2)
         )
 
+    def test_get_sorted_funcs(self):
+        self.assertItemsEqual([k for (k, v) in self.fs._get_sorted_funcs()], ('a', 'b'))
```

Comment:
The type of return value of `Function._get_sorted_funcs()` is an array of tuple(especially pair). So do you intended to write `tuple(self.fs._get_sorted_funcs()).keys()` instead?